src/app/directives/prev.directive.ts

```diff
@@ -0,0 +1,15 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({ selector: '[prev]' })
+export class PrevActivityDirective {
+    constructor(el: ElementRef) {
+        el.nativeElement.style.width = '75px';
+        el.nativeElement.style.lineHeight = '75px';
+        el.nativeElement.style.textAlign = 'center';
+        el.nativeElement.style.opacity = 1;
+        el.nativeElement.style.border = 'none';
+        el.nativeElement.style.color = 'white'
+        el.nativeElement.style.borderRadius = '50%';
+        el.nativeElement.style.background = '#D9534F';
```

Comment:
this is awfully similar to the `next.directive.ts` with just a difference in background color.
not to mention styling should typically done via css.